compiler/src/dotty/tools/dotc/ast/Positioned.scala

```diff
@@ -25,11 +28,8 @@ abstract class Positioned(implicit @constructorOnly src: SourceFile) extends Pro
 
   def uniqueId_=(id: Int): Unit = {
     if (Positioned.debugId == id) {
-      def printTrace() = {
-        val stack = Thread.currentThread().getStackTrace().map(">   " + _)
-        System.err.println(stack.mkString(s"> Debug tree (id=${Positioned.debugId}) creation \n> $this\n", "\n", "\n"))
-      }
-      printTrace()
+      println(s"Debug tree (id=${Positioned.debugId}) creation \n$this\n")
```

Comment:
Keep this code in the function `printTrace` or with another name. `uniqueId_=` is called a lot and we do not want to increase the size of this method for the benefit of the JIT